### ISSUE TITLE
Visibility of all requests of TA

### ISSUE THREAD

**Issue by mmarschler** [DiSSCo/ELViS#170]
In our understanding, it is planned that all requesters can see all submitted TA applications, as it is in VA requests. This is a function that is reasonable in VA, but - in our opinion - doesn't make sense in TA, as TA is very competitive.

**Comment by scottwilson-nhm** [comment]
Thank you Maria, this is indeed the case. 

@cannedit, apologies if this has been missed within the discussions we've had so far. VA has a collaborative, research-community focus so there is some reason why VA requests should be visible. 

TA is a more competitive pool and Requests ready to be evaluated should only be visible to admin /TAF Admin level. 

(And, of course, Scorers, when their functionality is implemented)

**Comment by cannedit** [comment]
I am sorry, but this is the result of combining VA and TA functionality and opening up all VA Requests was a specific request from the Synthesys+ WP6 community. Changing this concept at this stage of the development - while we are under a lot of pressure to deliver in time - is not possible anymore, because it will have a lot more repercussions than simply altering a filtering.

I understand that TA might have a different playing field compared to VA, but look at it this way: this attitude will have to change soon anyway because of the European Commission's efforts regarding "Open Science".

**Comment by wouteraddink** [comment]
This is a showstopper issue. TA calls are highly competitive, we cannot reveal names and proposals to everyone registered. It has be been stated very clearly and repeatedly in the design process for TA implementation that TA is very different from VA in this respect. I will schedule a meeting to discuss options how to deal with this issue given the time that is left.

**Comment by cannedit** [comment]
In the first version of ELViS we had all VA Requests only visible for persons involved: the requesters and VA coordinators, but that had to be changed: we had to allow everybody to take a look at those requests, so without any other instruction we could only assume that the same would go for the TA Requests. I understand that this has to change, but this might mean that we will get questions about the difference between this behaviour for TA and VA Access, so my question: should we also change this (so change this back) for VA Access?

**Comment by wouteraddink** [comment]
No because VA Requests are collaborative (requesters get no money, digitisation provided as community service), while TA Requests are competitive (requester gets money, best proposals win, they can be seen as bids for business).

**Comment by cannedit** [comment]
This has been fixed now (TA requests are only visible for people directly involved, so not for anyone else, no matter what their status is), so please test.

**Comment by jgrieb** [comment]
Tested this and works: As a requester I can see all VA requests but only the TA requests I submitted. As a TA coordinator I can see only these TA requests which were submitted to my institution, i.e. assigned to me.

Update: I just saw that TA requests from a different requester account assigned to me as TA coordinator were shown under the option "Requests assigned to me", but not under "All requests". So in the screenshot below the two requests from "Jonas Grieb Account2" do not appear under "All requests". I don't know if this on purpose but I think "All requests" should also display these.

![ta_reqests_filtering](https://user-images.githubusercontent.com/38728151/108351825-4e990500-71e6-11eb-8720-5207dbd385b9.png)
